nbconvert/filters/markdown.py

```diff
@@ -54,7 +55,10 @@ def markdown2latex(source, markup='markdown', extra_args=None):
     out : string
       Output as returned by pandoc.
     """
-    return convert_pandoc(source, markup, 'latex', extra_args=extra_args)
+    return convert_pandoc(source, markup, 'latex',
+        extra_args=extra_args,
+        relative_path_replacement=relative_path_replacement,
```

Comment:
I would not add these keyword arguments to `convert_pandoc`. I think it would be cleaner to do it in either: 
1. a jinja filter that is run prior to calling `convert_pandoc` 
2. a preprocessor that changes these links before jinja templating is applied 

going into the details of this would be too long for this little comment, but I'll add more details about the implications of both later.

The same point should be made for `build_path_replacement=build_path_replacement` below.